### PR TITLE
Repository templates for type

### PR DESCRIPTION
**Pull request by g4s8**
#286 - show repository templates for different repository types, added type switch.

### CODE REVIEW

File: src/main/resources/dashboard/repo.hbs [hunk 1]
```diff
@@ -71,6 +69,55 @@ only be able to download them. Nobody else will have any access to the repo.</p>
 <a href="https://maven.apache.org/plugins/maven-deploy-plugin/usage.html"><code>mvn deploy</code></a>
 and you download with
 <a href="https://maven.apache.org/plugins/maven-compiler-plugin/index.html"><code>mvn compile</code></a>.
+{{/eq}}{{#eq type "maven-proxy"}}
+<p>It's maven proxy repository, it proxies all request to remote Maven repository apply authentication
+and save artifacts in Artipie storage cache.</p>
```
Comment (olenagerasimova) [inline src/main/resources/dashboard/repo.hbs:74]: @g4s8 I'd suggest to rephrase:
Maven proxy repository proxies all request to remote Maven repository appling authentication
and saves artifacts in Artipie storage cache.

File: src/main/resources/dashboard/repo.hbs [hunk 2]
```diff
@@ -71,6 +69,55 @@ only be able to download them. Nobody else will have any access to the repo.</p>
...
+for <code>alpine:3.11</code> use:
+<pre>
+docker tag alpine:3.11 central.artipie.com/{{user}}/{{repo}}alpine:3.11
+docker push central.artipie.com/{{user}}/{{repo}}alpine:3.11
+</pre>
+</p>
+{{/eq}}
+{{#eq type "maven-group"}}
+To use Maven groupped (virtual) repository, you have to create at least two Maven repositories before,
+it can be either just a Maven repository or Maven proxy (mirror) repository. Then specify
```
Comment (olenagerasimova) [inline src/main/resources/dashboard/repo.hbs:104]: @g4s8 I'd suggest to rephrase:
To use Maven grouped (virtual) repository first create at least two Maven or Maven proxy repositories.

File: src/main/resources/dashboard/repo.hbs [hunk 3]
```diff
@@ -71,6 +69,55 @@ only be able to download them. Nobody else will have any access to the repo.</p>
...
+<code>https://central.artipie.com/{{user}}/{{repo}}</code> as distribution management, or
+repositories in <code>pom.xml</code> or as a mirror in <code>settings.xml</code> (see
+more details in concrete repositories documentation). Then group repositories under
+group-settings, pay attention that order does matter: first repository in the list will be
+accessed first, then second, etc. Repository name should be full formatted, include your username
```
Comment (olenagerasimova) [inline src/main/resources/dashboard/repo.hbs:109]: @g4s8 should be **fully** formatted

File: src/main/resources/dashboard/user.hbs [hunk 4]
```diff
@@ -1,22 +1,35 @@
 {{#partial "content"}}
 
 <p>
-  You may reset your password to be used in
-  authentication through package managers, like Maven:
+  To start working with Artipie using package managers you have to obtain
+  GitHub <a href="https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token">personal access token</a> to use it as a password. You username will be <code>github.com/{{user}}</code>.
```
Comment (olenagerasimova) [inline src/main/resources/dashboard/user.hbs:5]: @g4s8 hm, I think we should simplify this sentence:
Obtain GitHub personal access token and use it as a password in authentication through package managers, your username is github.com/{{user}}.

**Comment by g4s8** [comment]
@olenagerasimova thanks, all fixed.

**Comment by 0crat** [comment]
Job was finished in 13 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**